venue.html.j2

```diff
@@ -15,6 +15,24 @@
     </small>
   </figure>
 
+  <h2>Hotels</h2>
+
+PGConf.dev has negotiated discounted rates with several hotels in Montreal. We recommend that you book your accommodations early, whether or not you're staying at one of the hotels below, as there are many activities around Montreal at this time of the year.
+
+  <h3>Alt Hotel Montreal</h3>
+  <p>
+  <a href="https://www.germainhotels.com/en/le-germain-hotel/montreal">The Alt Hotel</a> is located
+  at 120, rue Peel, approximately a 15 minute walk from the conference venue.  Rooms are available
+  starting at $239 CAD a night.  You can book with this <a href="https://reservation.germainhotels.com/ibe/details.aspx?propertyid=17532&nights=1&checkin=05/12/2025&group=2505SQLDEV&lang=en-us&adults=1&childAges=">link</a>
+  Bookings with the above link must be made by April 12, 2025.  It might be possible to make bookings by phone or email using the code `2505SQLDEV` after this date
```

Comment:
Instead of backticks, use the HTML element [`<kbd>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd), which semantically denotes keyboard input. This has the benefit of being styled in monospace (once I integrate my new stylesheet).

Reply:
- [X] Added krb